package-lock.json

```diff
@@ -9,8 +9,6 @@
       "version": "0.1.0",
       "dependencies": {
         "@reduxjs/toolkit": "^1.9.3",
-        "@types/node": "18.15.11",
```

Comment:
remove this file from your changes as you have not installed any new package.

Reply:
okay, I will do just day